crates/holochain/src/conductor/handle.rs

```diff
@@ -208,14 +209,21 @@ pub trait ConductorHandleT: Send + Sync {
     /// Prune expired agent_infos from the p2p agents database
     async fn prune_p2p_agents_db(&self) -> ConductorResult<()>;
 
-    /// Create a new Cell in an existing App based on an existing DNA
+    /// Create a new cell in an existing app based on an existing DNA.
+    ///
+    /// # Returns
+    ///
+    /// An struct with the created cell's clone id and cell id.
     async fn create_clone_cell(
         self: Arc<Self>,
         payload: CreateCloneCellPayload,
-    ) -> ConductorResult<CellId>;
+    ) -> ConductorResult<InstalledCell>;
 
     /// Destroy a cloned Cell
```

Comment:
Assuming this is the meaning of the bool returned?

```suggestion
    /// Destroy a cloned Cell. Returns `true` if there was actually a clone to destroy, `false` if not.
```

Reply:
yes, but this was actually not supposed to be part of this PR.